content/quickstart/erlang/tracing.md

```diff
@@ -114,27 +159,46 @@ We will be tracing the execution as it starts in `read_eval_process/0`, goes to
 
 To accomplish this, we must create a span in each of the functions.
 
-You can create a span with `ocp:with_child_span/1` and finish with `ocp:finish_span/0`:
+You can create a span with the `ocp` module's function `with_child_span/1` and finish with `finish_span/0`, with `opencensus_elixir` a macro `with_child_span` is provided as well:
 
-```erlang
+{{<tabs Erlang Elixir>}}
+{{<highlight erlang>}}
 ocp:with_child_span(SpanName),
 ...
 ocp:finish_span().
-```
+{{</highlight>}}
 
-When creating a new span with `ocp:with_child_span/1`, the function first checks if a parent Span already exists in the process dictionary. If it exists, a child span is created. Otherwise, a newly created span is inserted in to the process dictionary to become the parent Span.
+{{<highlight elixir>}}
+with_child_span do
+  ...
+end
+{{</highlight>}}
+{{</tabs>}}
+
+
+When creating a new span with this way OpenCensus first checks if a parent Span already exists in the process dictionary. If it exists, a child span is created. Otherwise, a newly created span is inserted in to the process dictionary to become the parent Span.
 
 ## Exporting traces
 
 <a name="exporting-to-zipkin"></a>
 ### Import packages
 To enable exporting of traces to Zipkin, we'll need to set it as the reporter in `sys.config`:
 
-```erlang
+{{<tabs Erlang Elixir>}}
+{{<highlight erlang>}}
 [{opencensus, [{sampler, {oc_sampler_always, []}},
-               {reporter, {oc_reporter_zipkin, []}}]}
+               {reporters, [{oc_reporter_zipkin, []}]}]}
 ].
-```
+{{</highlight>}}
+
+{{<highlight elixir>}}
+use Mix.Config
+
+config :opencensus,
+  sampler: {:oc_sampler_always, []},
+  reporter: {:oc_reporter_zipkin, []}
```

Comment:
isn't it reporters now?

Reply:
Woops, yup, thought I fixed that already.

Reply:
fixed